ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I'm pretty sure you can delete this file. It will take the defaults from AgnosticD then. Also this file makes the PR check fail.

Reply:
Removed